xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1461,6 +1461,20 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
     stream->flags = (CDemuxStream::EFlags)pStream->disposition;
 
     AVDictionaryEntry *langTag = av_dict_get(pStream->metadata, "language", NULL, 0);
+    if (!langTag)
+    {
+      if ((strcmp(m_pFormatContext->iformat->name, "avi") == 0) && (streamIdx > 0 && streamIdx < 10))
```

Comment:
According to http://www.sno.phy.queensu.ca/%7Ephil/exiftool/TagNames/RIFF.html#Info, only "IAS1" to "IAS9" are specified, which correspond to stream 1 to 9. I'm not sure wether there is something for stream 0 (video stream).